uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/dialog/MessageDialog.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.fuse.reddeer.dialog;
+
+import org.eclipse.jface.window.Window;
+import org.eclipse.reddeer.core.handler.CompositeHandler;
+import org.eclipse.reddeer.jface.window.AbstractWindow;
+import org.eclipse.reddeer.jface.window.Openable;
+import org.eclipse.reddeer.swt.impl.label.DefaultLabel;
+import org.eclipse.swt.widgets.Label;
+
+public class MessageDialog extends AbstractWindow {
+	
+	private CompositeHandler handler = CompositeHandler.getInstance();
+	
+	public MessageDialog(String shellTitle) {
+		super(shellTitle);
+	}
+
+	@Override
+	public Openable getDefaultOpenAction() {
+		return null;
+	}
+
+	public String getMessage() {
+		checkShell();
+		// Label message is the second Widget
+		return new DefaultLabel(((Label)handler.getChildren(getShell().getSWTWidget())[1])).getText();
```

Comment:
I recomment to use a reference composite when looking for a widget + using index in the constructor, e.g
```
return new DefaultLabel(this, 1).getText();
```